src/DotNet.Status.Web/Program.cs

```diff
@@ -7,35 +7,37 @@
 using Microsoft.AspNetCore.Hosting;
 using Microsoft.DncEng.Configuration.Extensions;
 using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Hosting;
 using Microsoft.Extensions.Logging;
-using IHostingEnvironment = Microsoft.Extensions.Hosting.IHostingEnvironment;
 
 namespace DotNet.Status.Web
 {
     public static class Program
     {
         public static void Main(string[] args)
         {
-            new WebHostBuilder()
-                .UseKestrel()
+            Host.CreateDefaultBuilder()
+                .ConfigureWebHostDefaults(host =>
+                {
+                    host.UseStartup<Startup>()
+                        .UseUrls("http://localhost:5000/")
```

Comment:
Is this intended to be localhost url?

Reply:
This is just moving line 37 to here.  The three methods don't work on the "CreateDefaultBuilder", they have go inside the "ConfigureWebHostDefaults" now.